package.json

```diff
@@ -44,8 +45,5 @@
   "dependencies": {
     "@babel/runtime": "^7.12.13",
```

Comment:
why `@babel/runtime` in `dependencies` ?

Reply:
supporting new feature of js in the old environment without polluting global object, I think

Reply:
I think if we need `core-js` as a helper, but if have break we can go with it 
![image](https://user-images.githubusercontent.com/16365952/108417612-c60c7a00-7262-11eb-80f7-5b3d187026d5.png)

Reply:
idk if we need core-js right now, it has a lot of polyfills, I think for now it's ok to only use `@babel/runtime` and `@babel/plugin-transform-runtime`